x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/waffle/node_square.tsx

```diff
@@ -178,15 +183,21 @@ export const NodeSquare = ({
   });
   const style: CSSProperties | undefined = showBorder ? { border: 'solid 4px #000' } : undefined;
 
+  const onNodeSquareKeyDown = useCallback(
+    (event: KeyboardEvent) => {
+      if (event.key === keys.SPACE) {
+        togglePopover();
+      }
+    },
+    [togglePopover]
+  );
+
   return valueMode || ellipsisMode ? (
     <NodeContainer
       data-test-subj="nodeContainer"
       style={{ width: squareSize || 0, height: squareSize || 0 }}
       onClick={togglePopover}
-      onKeyPress={togglePopover}
-      onFocus={showToolTip}
-      onMouseOver={showToolTip}
-      onMouseLeave={hideToolTip}
```

Comment:
Removing this causes an issue in the host waffle map ( The tooltip should not be visible if the user moves the mouse out of the square - also it overlaps the opened flyout in some cases when it stays there ): 
Now (after this change)

https://github.com/user-attachments/assets/84badcd2-765c-421a-bf8f-91589d08d1c1


Before

https://github.com/user-attachments/assets/371d8ae6-0bcd-4d6a-a203-71c47e5d2be5

Another issue we can see here is the blue border (disappeared before the changes and now I see a double border when I click it - the black one and a blue one - it should be only the black one like in the `Before` example.

Reply:
**fixed**, video was attached in the description